docs/kubernetes/ingress-controller.md

```diff
@@ -281,3 +281,119 @@ Example ingress:
           - backend:
               serviceName: app-svc
               servicePort: 80
+
+## Install Skipper with enabled RBAC
+
+If Role-Based Access Control ("RBAC") is enabled you have to create some additional
+resources to enable Skipper to query the Kubernetes API.
+
+This guide describes all necessary resources to get Skipper up and running in a
+Kubernetes cluster with RBAC enabled but it's highly recommended to read the
+[RBAC docs](https://kubernetes.io/docs/admin/authorization/rbac/) to get a better
+understanding which permissions are delegated to Skipper within your Kubernetes cluster.
+
+First create a new `ServiceAccount` which will be assigned to the Skipper pods:
+
+    apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+      name:  aws-ingress-controller
```

Comment:
Why this is called aws-ingress-controller?
Skipper can perfectly run in any cluster type.

Reply:
Renamed to `skipper-ingress`